Coolbooru/Properties/AssemblyInfo.cs

```diff
@@ -31,5 +31,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("2.0")]
-[assembly: AssemblyFileVersion("2.0")]
+[assembly: AssemblyVersion("2.0.*")]
```

Comment:
Apologies for not replying sooner.

The * is replaced by the build and revision as determined by MSBuild. They're random(?) values between 0-65534, so Nuget would see them as ex 2.0.342.12332 (format: major.minor.build.revision).